### PR TITLE
Refactor global styles directory

### PR DESCRIPTION
**Pull request by mythilytm**
Primary reviewer:

## Description
- This PR introduces refinements to the global styles directory to move away from using the `HrmStyles` file. 
- Refactored all components from `HrmStyles`, excluding form elements. These components were either moved to their respective components level directory or if used in several places, moved to one of the following global styles modules:
  - buttons
  - layout
  - typography

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Related Issues
Fixes #....

### Verification steps
- Ensure features are working in the flex instance.

### CODE REVIEW

File: plugin-hrm-form/src/components/case/CaseTags.tsx [hunk 1]
```diff
@@ -20,8 +20,8 @@ import React from 'react';
 import { View, Text } from '@react-pdf/renderer';
 import { DefinitionVersion } from 'hrm-form-definitions';
 
-import { Flex } from '../../styles/HrmStyles';
-import { TagsWrapper } from '../../styles/search';
+import { Flex } from '../../styles';
+import { TagsWrapper } from '../search/styles';
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/CaseTags.tsx:24]: For a next PR: if this is shared maybe is worth moving upper in the tree?

File: plugin-hrm-form/src/components/case/CloseCaseDialog.tsx [hunk 2]
```diff
@@ -19,9 +19,9 @@ import { Template } from '@twilio/flex-ui';
 import WarningIcon from '@material-ui/icons/Warning';
 
 import TabPressWrapper from '../TabPressWrapper';
-import { Box, Row, StyledNextStepButton, HiddenText } from '../../styles/HrmStyles';
-import { CloseButton, CloseTaskDialog } from '../../styles/callTypeButtons';
-import { CloseDialogText } from '../../styles/case';
+import { Box, Row, HiddenText, StyledNextStepButton } from '../../styles';
+import { CloseButton, CloseTaskDialog } from '../callTypeButtons/styles';
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/CloseCaseDialog.tsx:23]: Same comment

File: plugin-hrm-form/src/components/case/ConnectToCaseButton.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ import { CheckCircleOutlineOutlined, CreateNewFolderOutlined } from '@material-u
 import { Template } from '@twilio/flex-ui';
 import React from 'react';
 
-import { PreviewActionButton } from '../../styles/search';
+import { PreviewActionButton } from '../search/styles';
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/ConnectToCaseButton.tsx:21]: :point_up_2:

File: plugin-hrm-form/src/components/case/ViewCaseItem.tsx [hunk 4]
```diff
@@ -21,8 +21,9 @@ import { Template } from '@twilio/flex-ui';
 import Edit from '@material-ui/icons/Edit';
 import { DefinitionVersion, isNonSaveable } from 'hrm-form-definitions';
 
-import { BottomButtonBar, Box, StyledNextStepButton } from '../../styles/HrmStyles';
-import { CaseLayout, FullWidthFormTextContainer } from '../../styles/case';
+import { BottomButtonBar, Box } from '../../styles';
+import { StyledNextStepButton } from '../../styles/buttons';
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/ViewCaseItem.tsx:25]: Pr another PR: if `styles/buttons` is exported in the `styles/index` you could collapse this import everywhere :thinking:

**Review by GPaoloni — APPROVED** [review]
Thanks!! This is pleasing to see :raised_hands: :rocket:

File: plugin-hrm-form/src/components/case/CaseTags.tsx (hunk 1 above)
Comment (GPaoloni) [reply plugin-hrm-form/src/components/case/CaseTags.tsx:24]: This occurs several times. Maybe we could improve that next: the places where we import from ../something/styles could means that those are actually shared styles.

Comment (mythilytm) [reply plugin-hrm-form/src/components/case/CaseTags.tsx:24]: Sounds good

**Comment by nickhurlburt** [comment]
Thanks, @mythilytm, this looks great. I want to add a couple things to our eng chat meeting topic list from this: 1. getting everyone caught up on the changes, 2. do we need to edit our coding standards to be in sync?

**Pull request merged**